### PR TITLE
Incident containers

### PR DESCRIPTION
**Pull request by percworld**
### What Changed?
- Resets App folder and deletes app
- Adds stylings to incidentsContainers
- Sets footer appropriately
- inserts scalable icons for incidents
- creates utilities file and formats date displayed with an export

### What's Next?
-Rednering of icons dynamically
### Known Issues?
- None. The hope is that with the working directory clear off the app conflict, we can move forward with rebasing more consistently.
### Test Coverage?
- All tests pass but we will move into testing all components with our backend responses.

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          }} />
+        </Switch>
+        <Footer />
+      </div>
+    </main>
+  )
+
+}
+
+export default App
```
Comment (JeffShepherd) [inline src/components/App/App.js:41]: Do we still have two versions of App in components and Components folders?

Comment (percworld) [reply src/components/App/App.js:41]: It appears that this extra Components folder may only be on your local machine.  Let's do a bit more research on that on Cameron and I's end.  This PR does not have 2 folders for components. So I'm thinking hopefully a reset on your end is a solution.

**Review by percworld — COMMENTED** [review]
Deleted duplicate Component folder using gitHub UI.  The remote repo now shows only compontents folder.

File: src/components/App/App.js (hunk 1 above)
Comment (cbmackintosh) [reply src/components/App/App.js:41]: Really weird behavior here - upon examining the remote branch on GitHub, we did discover a duplicate folder called Components containing App.js and App.css. However, that folder was not visible on our editors when we pulled the branch down locally. So we ultimately resorted to deleting the the duplicate files from the diff view in GitHub's UI. Hopefully this solves the problem

File: src/utilities.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const formatDate = inputDate => {
+  const monthNames = ["January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"];
+  let date = new Date(inputDate);
+  return monthNames[date.getMonth()] + ' ' + date.getDate() + ', ' + date.getFullYear();
+}
+
+module.exports = { formatDate };
```
Comment (cbmackintosh) [inline src/utilities.js:8]: Good call writing a helper function to format our dates. This file is a good place for it too.

File: src/components/OngoingIncidents/OngoingIncidents.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import React, { useEffect } from 'react'
 import { Link } from 'react-router-dom'
 import { useSelector, useDispatch } from 'react-redux'
+import { formatDate } from '../../utilities'
 import {
   getOngoingIncidents
 } from './OngoingIncidentsSlice'
 import './OngoingIncidents.css'
+import { ReactComponent as Fire } from '../../Icons/gripfire-brands.svg'
```
Comment (cbmackintosh) [inline src/components/OngoingIncidents/OngoingIncidents.js:10]: Sounds like you're working on a way to make these icon imports dynamic for the different incident types. The icons look really cool and add nicely to the user experience.

**Review by cbmackintosh — APPROVED** [review]

**Pull request merged**